yt/loaders.py

```diff
@@ -925,6 +925,7 @@ def load_octree(
     periodicity=(True, True, True),
     over_refine_factor=None,
     num_zones=2,
+    domain_dimensions=None,
```

Comment:
adding new arguments in between two existing ones for which positional values are accepted is a breaking change, please move it to after the `*,`